cove_ocds/urls.py

```diff
@@ -10,7 +10,10 @@
 
 
 # Serve the OCDS validator at /validator/
-urlpatterns_core += [url(r'^data/(.+)$', cove_ocds.views.explore_ocds, name='explore')]
+urlpatterns_core += [
+    url(r'^data/(.+)$', cove_ocds.views.explore_ocds, name='explore'),
+    url(r'^raw/(.+)$', cove_ocds.views.explore_ocds, name='explore_raw')
```

Comment:
Hmmm, two URLs mapping  to the same view. I'm not sure, it may be user cases for this, but I don't see it here. I understand that you are doing this so in the view you check for the URL with ```raw``` in it, but I don't think that is the best way to do so. If you want a view behaving differently for different requests, that is normally done using GET and POST parameters  https://docs.djangoproject.com/en/1.11/ref/request-response/

Reply:
@edugomez, this makes sense. Should have treated this in the same way as 'source_url' in GET (in fact, there's already a raw parameter sent with the url).